### ISSUE TITLE
build of VIEW fails with missing ui_assignBondOrderConfigurationDialog.h

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#516]
**Reported by friededn on 2 Nov 43866367 04:27 UTC**
I have tired to build the VIEW component of BALL and it keeps failing with:
In file included from /home/friedel/Downloads/BALL-1.4.2/source/VIEW/DIALOGS/assignBondOrderConfigurationDialog.C:5:
/BALL-1.4.2/include/BALL/VIEW/DIALOGS/assignBondOrderConfigurationDialog.h:21:65: error: BALL/VIEW/UIC/ui_assignBondOrderConfigurationDialog.h: No such file or directory
 and I am unsure how to generate/find the missing header file (I am building version 1.4.2)

**Comment by anhi** [comment]
**Commented by friededn on 20 May 43867781 07:00 UTC**
I was able to solve this by manually running uic-qt4 on all .ui files in source/VIEW/DIALOGS but a fix for this in the build would be nice. I also had to add -lboost_system to the BALL.dir/link.txt file in order to get the VIEW build to complete and -lboost_iostreams to the VIEW.dir/link.txt file to get BALLView to completely build. Boost was properly detected by the CMake run, so I do not know why I had to add these manually.

Scientific linux 6.4
kernel 2.6.32
gcc version 4.4.7
cmake version 2.8.12.1
